app/assets/stylesheets/my_modules/protocols/index.scss

```diff
@@ -322,9 +327,16 @@
   }
 
   .task-assigned-users {
-    height: 33px;
+    align-items: center;
+    border: 1px solid transparent;
+    display: flex;
+    height: 36px;
+    line-height: 34px;
+    padding: 2px;
```

Comment:
you don't need line-height, when you use aling-items.
Add width same as height, so it will be square.
And justify content to center.
And after this you can remove padding.